docs/src/ho.md

```diff
@@ -53,7 +53,7 @@ plot(fNC,-1., 1, label="fNC")
 
 
 ```@example main
-@def ocp begin
+  @def ocp begin
```

Comment:
At the end of the code, you can add `nothing # hide` if you want.

Reply:
Thanks, I used `nothing # hide` for most of the code and it worked well except in
```@example main
N = 500
sol = solve(ocp; grid_size=N); 
nothing # hide
```
where it (still) shows the output.

Reply:
This is not really an output but a trace during the optimization process. To turn it off you can use the option `display`:

```julia
N = 500
sol = solve(ocp; grid_size=N, display=false)
```

and if you don't want to print the output, here print `sol`, then you can add:

```julia
N = 500
sol = solve(ocp; grid_size=N, display=false)
nothing # hide
```

**Note.** You can remove ";" when using `nothing # hide`.

Reply:
Noted, Thanks!

Reply:
If you want the trace but there are many iterations and you want a shorter display, you can use the `print_level` option from `Ipopt`.

```julia
sol = solve(ocp; grid_size=N, print_level=4)
```